packages/react/src/infrastructure/theme/types.ts

```diff
@@ -0,0 +1,151 @@
+export enum LogoColor {
+  BLACK,
+  WHITE,
+}
+
+export enum ConnectWalletDefaultIconType {
+  WITH_COLOR,
+  WHITE,
+}
+
+export enum ThemeType {
+  DARK_MODE = 'DarkMode',
+  LIGHT_MODE = 'LightMode',
+}
+
+export type ThemeContextType = {
+  darkModeFsEnabled: boolean;
+  theme: ThemeType;
+  toggleTheme: () => void;
+};
+export interface Theme {
+  images: {
+    logoImageColor: LogoColor;
+    connectWalletInactiveIconType: ConnectWalletDefaultIconType;
+  };
+  colors: {
+    beatLoader: string;
+
+    cssColorScheme: 'dark' | 'light'; // use only for css color-scheme value
```

Comment:
Can these comments be removed? Do we support dark mode on react button? (if not we can remove all of the dark mode support here).